create.ts

```diff
@@ -0,0 +1,803 @@
+import question from "https://raw.githubusercontent.com/ocpu/question-deno/master/mod.ts";
```

Comment:
Is the API this provides using stable Deno features? We'd like to avoid having to add --unstable